src/Microsoft.Health.Fhir.Liquid.Converter/Models/Json/JsonTraceInfo.cs

```diff
@@ -0,0 +1,24 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Newtonsoft.Json.Schema;
+
+namespace Microsoft.Health.Fhir.Liquid.Converter.Models.Json
+{
+    public class JsonTraceInfo : TraceInfo
```

Comment:
Do we need to update `ConverterLogicHandler.cs` in converter tool? Initialize JsonTraceInfo when type is Json.
Another question: Is `ValidateSchemas` only applicable for type Json?